PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingViewController.swift

```diff
@@ -47,28 +47,47 @@ class PetListingViewController: UIViewController
         self.collectionView.dataSource = self
         self.collectionView.alwaysBounceVertical = true
         self.collectionView.backgroundColor = UIColor.groupTableViewBackgroundColor()
-        self.collectionView.collectionViewLayout = CustomHomeCollectionViewFlowLayout()
+        self.collectionView.collectionViewLayout = HomePortraitCollectionViewLayout()
         
         refreshControl.addTarget(self, action: #selector(refreshTriggered), forControlEvents: .ValueChanged)
         self.collectionView.addSubview(refreshControl)
         
         loadPets()
-
     }
 
+    ////////////////////////////////////////////////////////////
+
     override func viewWillAppear(animated: Bool)
     {
         super.viewWillAppear(animated)
         self.navigationItem.title = NSLocalizedString("Town Of Lady Lake", comment: "")
     }
 
+    ////////////////////////////////////////////////////////////
```

Comment:
perhaps it is a personal preference but wouldn't be cleaner to only have them between sections?